<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CManual</class>
 <widget class="QWidget" name="CManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/common/manual.png);
</string>
   </property>
   <widget class="QLabel" name="mRgMsbNoHint">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>15</y>
      <width>30</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">10</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_1">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>15</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mYDimHint">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>90</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">23</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRgVal">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>48</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xAB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mYVal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>69</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xABCDE</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSVal">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>74</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xAB</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
